openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/storages/builders/AbstractGraphStorageBuilder.java

```diff
@@ -32,6 +33,8 @@ public void setParameters(Map<String, String> parameters)
 	{
 		this.parameters = parameters;
 	}
-	
+
 	public void finish(){}
+
+    public void postProcess(ORSGraphHopper graphHopper) {}
```

Comment:
Given the fact that this method is in the interface for a lot of classes but only ever implemented and used in the Here Storage, it would be better to simply type cast the storage in line 610 of `ORSGraphhopper`. Then you'd only have the postprocess method in the relevant storage and not cluttering anything else.

Reply:
Done